packages/manager/modules/ng-layout-helpers/src/onboarding/README.md

```diff
@@ -0,0 +1,51 @@
+# ng-ovh-onboarding
+
+> Display a empty page with details about the service to onboard user.
+
+## Install
+
+```sh
+yarn add @ovh-ux/manager-ng-layout-helpers
```

Comment:
for me, this command doesn't work. yarn say 'not found'

Reply:
however, is the case of other modules, this command should work ?

Reply:
@Heiiji I have updated the README.md with proper import statement.
import { OvhOnboarding } from '@ovh-ux/manager-ng-layout-helpers';